src/components/UserNameInputDialog.vue

```diff
@@ -18,21 +18,23 @@
           v-if="!isUsernamesEmpty && !showNewUsernamePrompt"
           class="w-full h-full flex flex-col align-center justify-center text-center"
         >
-          <p v-if="missionStore.username === undefined">
-            It seems like you don't have an username set in this device yet.
-          </p>
+          <p v-if="!missionStore.username">It seems like you don't have an username set in this device yet.</p>
```

Comment:
Any reason this was changed? Checking for undefinity is more explicit (and thus safer) than checking for nullish.